content/influxdb/cloud-iox/reference/sql/where.md

```diff
@@ -0,0 +1,129 @@
+menu:
+  influxdb_cloud_iox:
+    name: WHERE clause
+    parent: SQL reference
+weight: 202
+---
+
+Use the `WHERE` clause to filter results based on fields, tags, and/or timestamps.
+
+- [Syntax](#syntax)
+- [Examples](#examples)
+
+## Syntax
+
+```sql
+SELECT_clause FROM_clause WHERE <conditional_expression> [(AND|OR) <conditional_expression> [...]]
+```
+
+{{% note %}}
+**Note:** Unlike InfluxQL, SQL **supports** `OR` in the `WHERE` clause to specify multiple conditions, including time ranges.
+{{% /note %}}
+
+## Examples
+
+Note that single quotes are required for string literals in the `WHERE` clause. 
+
+### Filter data based on specific field value
+
+```sql
+SELECT * 
+FROM "h2o_feet" 
+WHERE "water_level" >= 9.78
+```
+
+{{< expand-wrapper >}}
+{{% expand "View example results" %}}
+
+The query returns data from the measurement h2o_feet with field values of water_level that are greater than or equal to 9.78. Note that this is a partial data set.
+
+| level description         | location     | time                     | water_level |
+| :------------------------ | :----------- | :----------------------- | :---------- |
+| at or greater than 9 feet | coyote_creek | 2019-09-01T23:06:00.000Z | 9.8         |
+| at or greater than 9 feet | coyote_creek | 2019-09-01T23:12:00.000Z | 9.829       |
+| at or greater than 9 feet | coyote_creek | 2019-09-01T23:18:00.000Z | 9.862       |
+| at or greater than 9 feet | coyote_creek | 2019-09-01T23:24:00.000Z | 9.892       |
+| at or greater than 9 feet | coyote_creek | 2019-09-01T23:30:00.000Z | 9.902       |
+| at or greater than 9 feet | coyote_creek | 2019-09-01T23:36:00.000Z | 9.898       |
+
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+
+### Filter data based on specific tag and field values
+
+```sql
+SELECT * 
+FROM "h2o_feet" 
+WHERE "location" = 'santa_monica' and "level description" = 'below 3 feet' 
+```
+
+{{< expand-wrapper >}}
+{{% expand "View example results" %}}
+
+The query returns all data from the `h2o_feet` measurement with the location tag key `santa_monica` and field value of `level description` that equals the `below 3 feet` string. This is a partial data set.
+
+| level description | location     | time                     | water_level |
+| :---------------- | :----------- | :----------------------- | :---------- |
+| below 3 feet      | santa_monica | 2019-09-01T00:00:00.000Z | 1.529       |
+| below 3 feet      | santa_monica | 2019-09-01T00:06:00.000Z | 1.444       |
+| below 3 feet      | santa_monica | 2019-09-01T00:12:00.000Z | 1.335       |
+| below 3 feet      | santa_monica | 2019-09-01T00:18:00.000Z | 1.345       |
+| below 3 feet      | santa_monica | 2019-09-01T00:24:00.000Z | 1.27        |
+
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+###  Filter data within a specific time period
+
+```sql
+SELECT *
+FROM h2o_feet 
+WHERE "location" = 'santa_monica'
+AND "time" >= '2019-08-19T12:00:00Z'::timestamp AND "time" <= '2019-08-19T13:00:00Z'::timestamp 
```

Comment:
Minor nit here: You don't have to cast these strings to timestamps because the query engine will recognize them as timestamps. The query is correct as-is, but if we wanted to be less verbose, we could.

Reply:
I'll keep as it's this way in all the reference examples but as we work on functions and add content I won't use the cast.